src/Container/Container.js

```diff
@@ -17,10 +17,16 @@ export default function Container({
   const [query, setQuery] = useState({});
 
   const querySetter = ({ nsValues }) => {
-    const filteredApplications = filterApplications(applicationsList, checkedAppIdsMap, nsValues.filters, nsValues.query);
+    const currentQuery = { ...query, ...nsValues };
```

Comment:
Could this be simplified to 
```
const currentQuery ={ query: undefined, ...query, ...nsValues };
```
and you could lose the conditional at 24?

Reply:
Doesn't quite work unfortunately. When clicking `Reset all`, the old search value is retained, so I need the conditional to clear the `query`.